2001-3000/LIT2629Zenahu.xml

```diff
@@ -35,18 +36,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage><language ident="en">English</language>
+            <language ident="gez">Gǝʿǝz</language></langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change who="DE" when="2017-07-13">added short title</change>
+         <change who="DR" when="2020-07-08">added short title and bibliography</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <ab/>
+         <div type="bibliography">
+            <listBibl type="secondary">
+               <bibl><ptr target="bm:Getachew2014ZenahuLagalla"/></bibl>
+               <bibl><ptr target="bm:Littmann1896Schleicher"/></bibl>
+            </listBibl>
+            <listBibl type="editions">
+               <bibl><ptr target="bm:ContiRossini1907SarsaDengel"/><citedRange unit="page">221-232</citedRange></bibl>
```

Comment:
There is a reasonable Russian translation of Zenahu la-Galla by Chernetsov, unfortunately not quoted in EAE article. It is published here: Chernetsov, Sevir B. Efiopskie xroniki XVI-XVII vekov. Übersetzt von Sevir B. Chernetsov. Moscow: Nauka Publishers, 1984.
I can provide the exact reference when I am back.